_api_app/app/Sections.php

```diff
@@ -5,7 +5,105 @@
 use App\Entries;
 use App\Tags;
 
+
+/**
+ * This class is a service that handles site section data for Berta CMS.
+ * Sections are stored in `sections.xml` file for the corresponding site.
+ *
+ * The root site has its sections stored in `storage/sections.xml`,
+ * any other site has it's sections in `storage/-sites/[site name]/sections.xml`
+ *
+ * @example an example of XML file:
+ * ```xml
+ * <?xml version="1.0" encoding="utf-8"?>
+ * <sections>
+ *   <section tags_behavior="invisible" published="1" entry_count="2" has_direct_content="0">
+ *     <name><![CDATA[first section]]></name>
+ *     <title><![CDATA[First Section]]></title>
+ *     <backgroundVideoEmbed><![CDATA[https://youtu.be/video]]></backgroundVideoEmbed>
+ *     <mediafolder><![CDATA[media-folder]]></mediafolder>
+ *     <mediaCacheData hide_navigation="yes" caption_bg_color="235,73,73" autoplay="0" image_size="small">
+ *       <file type="image" src="some-image.jpg" width="1200" height="640"><![CDATA[<p>A caption for this image</p>]]></file>
+ *     </mediaCacheData>
+ *   </section>
+ *   <section tags_behavior="invisible" published="1" has_direct_content="0">
+ *     <name><![CDATA[second section]]></name>
+ *   </section>
+ * </sections>
+ * ```
+ */
 class Sections Extends Storage {
+    /**
+     * @var array $JSON_SCHEMA
+     * Associative array representing data structure handled by this service.
+     */
+    public static $JSON_SCHEMA = [
+        '$schema' => "http://json-schema.org/draft-06/schema#",
+        'type' => 'array',
+        'items' => [
+            'type' => 'object',
+            'properties' => [
+                'name' => ['type' => 'string'],
+                'title' => ['type' => 'string'],
+                'backgroundVideoEmbed' => ['type' => 'string'],
+                'mediafolder' => ['type' => 'string'],
+                'mediaCacheData' => [
+                    'type' => 'object',
+                    'properties' => [
+                        'file' => [
```

Comment:
Actually there can be more than one file in background gallery, we should use `items` here.